### DISCUSSION TITLE
Residenza dell'Intestatario ignota.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#929] [Miglioramenti] [answered]
Buongiorno,

riportiamo una segnalazione dal Comune di Carpi:

E’ previsto che il cittadino intestatario dell’atto possa essere residente in Italia (con indicazione del comune e provincia di residenza) o all’estero (con indicazione dello stato).
A volte può succedere che la residenza dell’intestatario non è conosciuta o non è certificabile, ad esempio il cittadino è stato dichiarato irreperibile da un Comune italiano.
In questo secondo esempio nell’atto si indica come residenza “Irreperibile”. Come possiamo comunicare questa informazione?

In attesa di riscontro,

cordiali saluti.

**Comment by mttfranci** [comment, marked as answer]
Grazie per segnalazione @gcuci , abbiamo preso in carico la vostra osservazione e la valuteremo con il gruppo di analisi.

Saluti

Assistenza ANSC

**Comment by asqueglia** [comment]
Come da roadmap https://github.com/italia/ansc/blob/main/docs/Roadmap.md l'intervento è previsto per settembre.
